include/unicorn/arm64.h

```diff
@@ -313,6 +313,10 @@ typedef enum uc_arm64_reg {
 
     UC_ARM64_REG_PSTATE,
 
+    //> floating point control and status registers
```

Comment:
This will change the values of other enums which breaks compatibility. Please add the registers in the endings.